utilities/test_suite/HIP/Tensor_audio_hip.cpp

```diff
@@ -171,6 +175,77 @@ int main(int argc, char **argv)
 
                     break;
                 }
+                case 7:
+                {
+                    testCaseName = "mel_filter_bank";
+
+                    Rpp32f sampleRate = 16000;
```

Comment:
@HazarathKumarM @sampath1117 
Inside the test case for melfilter bank, we want something simple.
1. Please try to make all these initializations more concise and reduce number of lines.
2. There will be common code between host and hip. Please add a common init function in the rpp_test_suite_audio.h and use in both HOST and HIP -> like say init_ricap() or init_slice() here - https://github.com/r-abishek/rpp/blob/master/utilities/test_suite/HIP/Tensor_hip.cpp#L1349
3. This unsigned long long multiple times below in HOST and HIP is Rpp64u. Please use that.

Reply:
Done